js/ui/html_editor/modules/resizing.js

```diff
@@ -68,7 +69,14 @@ export default class ResizingModule {
 
     showFrame() {
         this._$resizeFrame.show();
-        eventsEngine.on(this.quill.root, KEYDOWN_EVENT, this.hideFrame.bind(this));
+        eventsEngine.on(this.quill.root, KEYDOWN_EVENT, this._handleFrameKeyDown.bind(this));
+    }
+
+    _handleFrameKeyDown(e) {
+        if(e.key === 'Delete' || e.key === 'Backspace') {
```

Comment:
Use `normalizeKeyName`, some old browsers can have a specific `key` value

```js
import { normalizeKeyName } from '../../../events/utils';
```

```suggestion
        const keyName = normalizeKeyName(e);
        if(keyName  === 'del' || keyName  === 'backspace') {
```